mender-update/context.hpp

```diff
@@ -158,6 +162,13 @@ class MenderContext {
 	conf::MenderConfig &config_;
 };
 
+// Only here to make testing easier, use MenderContext::MatchesArtifactDepends().
```

Comment:
Can't you then hide it in a namespace?

Reply:
I could, but it wouldn't make much difference.